lib/modules/accountSwitcher.js

```diff
@@ -106,10 +120,18 @@ module.options = {
 	},
 };
 
-let userLink, $downArrow, $downArrowOverlay;
+let userLink, $downArrow, $downArrowOverlay, queryParameter;
 
 export let $accountMenu;
 
+module.beforeLoad = () => {
+	if(isPageType('comments') && module.options.autoSwitchOnPost.value) {
```

Comment:
Instead of checking `if (isPageType('comments')...` you can simply add `module.includes = ['comments'];` in the "manifest" area above.

Reply:
Doing this causes each comment on a user page to be autoSwitched, eventually leading to you being logged out of reddit.